src/RemoteInboxNotifications/WooCommerceAdminUpdatedRuleProcessor.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Rule processor for sending when WooCommerce Admin has been updated.
+ */
+
+namespace Automattic\WooCommerce\Admin\RemoteInboxNotifications;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Rule processor for sending when WooCommerce Admin has been updated.
+ */
+class WooCommerceAdminUpdatedRuleProcessor implements RuleProcessorInterface {
+	/**
+	 * Process the rule.
+	 *
+	 * @param object $rule         The specific rule being processed by this rule processor.
+	 * @param object $stored_state Stored state.
+	 *
+	 * @return bool Whether the rule passes or not.
+	 */
+	public function process( $rule, $stored_state ) {
+		return get_option( RemoteInboxNotificationsEngine::WCA_UPDATED_OPTION_NAME, false );
```

Comment:
Should there be a version number check here?

For example, if there is a note highlighting WooCommerce 5.0 features in the future, then I think it would need to be skipped in cases when the store is only updated to a lesser version like 4.9.1.

Reply:
Good point, I'll have a think about how to handle that

Reply:
@adrianduffell This isn't needed because there is another rule that checks the version of a plugin. So to target upgrading specifically to WooCommerce 5 you would use the `woocommerce_admin_updated` rule plus the `plugin_version` rule.

Reply:
@becdetat That makes sense, but would be better if we can add that detail in RemoteInboxNotification readme? It wasn't 100% apparent to me the first time I see it

Reply:
Thanks @ilyasfoo , added in 95797dc

Reply:
Looks good!